SPECS/grpc/grpc.spec

```diff
@@ -93,26 +92,26 @@ pushd cmake/build
    -DgRPC_ZLIB_PROVIDER:STRING='package'
 %cmake_build 
 popd
-#python
-%ifarch x86_64
-   export GRPC_PYTHON_BUILD_WITH_CYTHON=True
-   export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
-   export GRPC_PYTHON_BUILD_SYSTEM_ZLIB=True
-   export GRPC_PYTHON_BUILD_SYSTEM_CARES=True
-   export GRPC_PYTHON_BUILD_SYSTEM_RE2=True
-   export GRPC_PYTHON_BUILD_SYSTEM_ABSL=True
-   %py3_build
-%endif
+#uncommenting below line causes the whole build to get stuck in aarch64 machine 
+#py3_build
```

Comment:
what's the effect of not having %py3_build in the %build section anymore (for x64)? Does it change any of the built files?

Reply:
there seems to be no effect for both x86_64 and aarch64.